packages/vm/lib/index.ts

```diff
@@ -89,6 +89,13 @@ export interface VMOpts {
    * Default: `false` [ONLY set to `true` during debugging]
    */
   allowUnlimitedContractSize?: boolean
+
+  /**
+   * Select hardfork based upon block number. This automatically switches to the right hard fork based upon the block number.
+   *
+   * Default: `true`
+   */
+  selectHardforkByBlockNumber?: boolean
```

Comment:
???

We can't having this default to `true`, this is a giga-breaking change.

Reply:
Yes you are right, will default this to `false` in a new PR.